### PR TITLE
Added documentation for custom inboxes

### PR DESCRIPTION
**Pull request by hendrik-hackerone**

### CODE REVIEW

**Review by DeRRudi77 — APPROVED** [review]

File: docs/organizations/custom-inbox-remove.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+id: "organizations/custom-inboxes-remove"
+---
+
+You can remove a custom inbox if you no longer need it.
+
+><i>Tip: Bear in mind that removing an inbox will revoke access to the reports for the groups that had access before.</i>
+
+You can remove an inbox from two places:
+
+1. Select Remove inbox from the action picker in the inbox overview.
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/custom-inbox-remove.md]: Yes it should 👍

File: docs/organizations/custom-inbox-remove.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+id: "organizations/custom-inboxes-remove"
+---
+
+You can remove a custom inbox if you no longer need it.
+
+><i>Tip: Bear in mind that removing an inbox will revoke access to the reports for the groups that had access before.</i>
+
+You can remove an inbox from two places:
+
+1. Select Remove inbox from the action picker in the inbox overview, or
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/custom-inbox-remove.md]: Removed the `, or`

File: docs/organizations/custom-inboxes-new.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+title: "Creating a new custom inbox"
+path: "/organizations/custom-inboxes-new.html"
+id: "organizations/custom-inboxes-new"
+---
+
+><i>Tip: Before creating a new custom inbox, [create the relevant groups](/organizations/groups.html) for the custom inbox. Use naming conventions that make it easy to map to your workflows and teams.</i>
+
+![custom inboxes new](./images/custom-inboxes-new.png)
+
+
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/custom-inboxes-new.md]: Removed the blank lines

File: docs/organizations/custom-inboxes.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+title: "Custom inboxes"
+path: "/organizations/custom-inboxes.html"
+id: "organizations/custom-inboxes"
+---
+
+Custom inboxes allow you to separate report access based on your teams, assets, or business units. Organization administrators can create additional inboxes to support their organization structures and workflow needs.
+
+As custom inboxes are not connected to an engagement, they can provide access to a subset of reports without granting access to the engagements themselves.
+
+
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/custom-inboxes.md]: Removed the blank lines

File: docs/organizations/custom-inboxes-new.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+1. Go to **Inboxes** in **Organization Settings**
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/custom-inboxes-new.md]: Added the period

File: docs/organizations/custom-inboxes-allocating.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Allocating a report to a custom inbox"
+path: "/organizations/custom-inboxes-allocating.html"
+id: "organizations/custom-inboxes-allocating"
+---
+
+After creating a custom inbox, users with report permissions in an engagement's inbox can allocate reports.
+
+1. Open the relevant report in the engagement inbox.
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/custom-inboxes-allocating.md:9]: I don't think so, as it doesn't refer to a text in the browser

[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/custom-inboxes-allocating.md:9]: But I'll check to make sure

**Comment by jessiwright** [comment]
Give me a few minutes to go through this. It was on my list to write this document this week, but I had to get the images approved first.

**Review by jessiwright — APPROVED** [review]

**Pull request merged**